Sources/MapboxCoreNavigation/TilesetDescriptorFactory.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import MapboxNavigationNative
+import MapboxDirections
+
+extension TilesetDescriptorFactory {
+    /**
+     Gets TilesetDescriptor which corresponds to current Navigator dataset and the specified `version`.
+     - parameter cacheLocation: A `Location` where cache data is stored.
+     - parameter version: TilesetDescriptor version.
+     */
+    open class func getSpecificVersion(forCacheLocation cacheLocation: TileStoreConfiguration.Location = .default, version: String) -> TilesetDescriptor {
```

Comment:
Maybe
```suggestion
    open class func getSpecificVersion(for cacheLocation: TileStoreConfiguration.Location = .default, version: String) -> TilesetDescriptor {
```
would be less cumbersome (similarly for second method in this extension)?

Reply:
Even though it's suboptimal, but other `TilesetDescriptorFactory` methods are mapped from Obj-C and look like `getSpecificVersion(forCache:, version:)`. I decided to repeat the pattern to blend the extension methods.